core/unit_test/TestTeam.hpp

```diff
@@ -1286,8 +1286,16 @@ struct TestTeamBroadcast<
     using policy_type_f =
         Kokkos::TeamPolicy<ScheduleType, ExecSpace, BroadcastTag>;
 
+    // FIXME_OPENMPTARGET temporary restriction for team size to be at least 32
+#ifdef KOKKOS_ENABLE_OPENMPTARGET
```

Comment:
Since execution spaces are unconditionally declared now, we shouldn't need this `ifdef` any more (right)?

Reply:
No, we haven't merged #3838.